include/emds/emds.h

```diff
@@ -71,8 +71,9 @@ struct emds_dynamic_entry {
  * @brief Callback for application commands when storing has been executed.
  *
  * This can be used to perform other actions or execute busy-waiting until the
- * power supply is discharged. It will run in the thread of emds. Scheduler and
- * interrupts are unlocked before triggering this callback. Therefore, this
+ * power supply is discharged. It will be run by @ref emds_store. It will be
+ * called from the same context which @ref emds_store is called from.
+ * Interrupts are unlocked before triggering this callback. Therefore, this
```

Comment:
```suggestion
 * power supply is discharged. It will be run by @ref emds_store,
 * and called from the same context the @ref emds_store is called from.
 * Interrupts are unlocked before triggering this callback. Therefore, this
```